redfish-core/lib/account_service.hpp

```diff
@@ -2325,11 +2553,18 @@ inline void
     std::optional<bool> locked;
     std::optional<std::vector<std::string>> accountTypes;
     std::optional<nlohmann::json> oem;
+    std::optional<std::vector<std::string>> mfaBypass;
 
-    if (!json_util::readJsonPatch(req, asyncResp->res, "UserName", newUserName,
-                                  "Password", password, "RoleId", roleId,
-                                  "Enabled", enabled, "Locked", locked, "Oem",
-                                  oem, "AccountTypes", accountTypes))
+    if (req.session == nullptr)
```

Comment:
If this check is moved here, this will negate L2572-L2577 code - https://github.com/ibm-openbmc/bmcweb/pull/824

I think this part L2558-L2561 should be deleted.

Reply:
Done